spec/user_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'user'
+
+describe User do
+  let(:diego) { User.new '+44 7496548640' }
+  let(:diego_fake) {double(:diego_fake, :place_food_order => true, :food_menu_food_order => {} ) }
```

Comment:
Line is too long. [98/80]<br>Use the new Ruby 1.9 hash syntax.<br>Space missing inside {.<br>Space inside parentheses detected.